index.css

```diff
@@ -8,14 +8,15 @@ body {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
 
 .wrapper {
   background-color: #1c1a19;
-  height: 100vh;
   background-image: url('./images/bg-header-mobile.png');
```

Comment:
- Hey, let's improve the placement of our image as seen in the design template below;


*YOUR DESIGN*
![Screenshot from 2023-01-14 13-03-17](https://user-images.githubusercontent.com/59397159/212470812-20ca39d9-3b26-4776-8749-d0bbe7bf22be.png)


*TEMPLATE*
![Screenshot from 2023-01-14 13-01-27](https://user-images.githubusercontent.com/59397159/212470815-f90b1a96-b9e2-44b0-bc68-ea4c2e396cd8.png)

Reply:
Hey @zieeco ,
thank you for the constructive feedback
I have made the requested changes and this is how it looks when I try it on the browser

![about-me](https://user-images.githubusercontent.com/77828236/212472579-5a6f6380-a577-456d-b3e4-8ea13612eff3.png)

Would you please check it once again?
Thank you